moped-editor/src/utils/activityLogHelpers.js

```diff
@@ -1,156 +1,19 @@
-    areLookups: false,
-    query: INITIAL_QUERY,
-  };
-
-  if (!tableNames || tableNames.length === 0) return noLookupsObject;
-
-  const tablesMap = {};
-
-  const lookupQueries = tableNames.map(tableName => {
-    const fields = ProjectActivityLogTableMaps[tableName].fields;
-
-    const lookupFields = Object.entries(fields).reduce(
-      (acc, [primaryFieldName, fieldConfig]) => {
-        return fieldConfig.hasOwnProperty("lookup")
-          ? [...acc, { ...fieldConfig.lookup, primaryFieldName }]
-          : [...acc];
-      },
-      []
-    );
-
-    // We need the field name from the original table here
-    const tableQueries = lookupFields.map(field => {
-      const { table, fieldLabel, fieldValues, primaryFieldName } = field;
-
-      const fieldValuesAliasesAndFieldNames = fieldValues
-        .map((fieldValue, i) => `value${i}: ${fieldValue}`)
-        .join(" ");
-
-      tablesMap[table] = tableName;
-
-      // Alias table name as primary table field name and return foreign key and value
-      return `
-      ${primaryFieldName}: ${table} {
-        key: ${fieldLabel}
-        ${fieldValuesAliasesAndFieldNames}
-      }
-      `;
-    });
-
-    return tableQueries;
-  });
-
-  const flatLookupQueries = lookupQueries.flat();
-
-  if (flatLookupQueries.length === 0) return noLookupsObject;
-
-  const lookupQueriesString = flatLookupQueries.join(" ");
-
-  const LOOKUPS_QUERY = gql`
-    query RetrieveLookupValues { 
-      ${lookupQueriesString} 
-    }
-  `;
-
-  return { areLookups: true, query: LOOKUPS_QUERY, tablesMap };
-};
-
-/**
- * Take activity log response and return unique table names for lookup in project's log
- * @param {object} response - GraphQL response containing project record types (table names)
- * @param {string} lookupDataKey - The key in the response whose values is the lookup data
- * @return {array} Array of unique table names for log data lookups
- */
-export const getActivityLogTableNames = (response, lookupDataKey) => {
-  const lookupTableData = response?.[lookupDataKey];
-
-  // would this length ever be more than one??
-  if (!lookupTableData || lookupTableData.length === 0) return null;
-
-  const tableNames = lookupTableData.map(record => record.record_type);
-
-  return tableNames;
+import { formatProjectActivity } from "./activityLogFormatters/mopedProjectActivity";
+import { formatTagsActivity } from "./activityLogFormatters/mopedTagsActivity";
+
+export const formatActivityLogEntry = (change, lookupData) => {
+  const changeDescription = "Project was updated";
+  const changeValue = "";
+  const changeIcon = (
+    <span className="material-symbols-outlined">summarize</span>
+  );
+
+  switch (change.record_type) {
+    case "moped_project":
+      return formatProjectActivity(change, lookupData.entityList);
+    case "moped_proj_tags":
+      return formatTagsActivity(change, lookupData.tagList);
+    default:
+      return { changeIcon, changeDescription, changeValue };
```

Comment:
i guess it'd be nice to  separate the default/fallback behavior into a `defaultFormatActivity` function?

Reply:
Definitely, if we want to be more detailed than everything else being the generic "Project was updated"